pygfx/helpers/_box.py

```diff
@@ -46,27 +61,34 @@ def __init__(self, size=1.0, thickness=1, color="white"):
             dtype="f4",
         )
         positions -= 0.5
+
+        # @almarklein: does this mean we currently don't support non-isotropic scale?
+        # otherwise we may want to update this to set the tranform's scale instead.
         positions *= self._size
 
         geometry = Geometry(positions=positions)
         material = LineSegmentMaterial(thickness=thickness, color=color, aa=True)
 
         super().__init__(geometry, material)
 
+    # should this become a classmethod? (alternative constructor)
```

Comment:
I don't think so. With this method you can have a single helper object, and use it to "highlight" objects in the scene, e.g. as the user clicks them.

Reply:
Ah, I see.